tests/test_playbooks/templates_import.yml

```diff
@@ -0,0 +1,59 @@
+---
+- hosts: localhost
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - include: tasks/provisioning_template.yml
+      vars:
+        provisioning_template_name: "fam_tsync_test_Kickstart default"
+        provisioning_template_kind: provision
+        provisioning_template_state: absent
+
+- hosts: tests
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+    - vars/templates_import.yml
+  tasks:
+    - include_tasks: tasks/templates_import.yml
+      vars:
+        templates_import_prefix: "fam_tsync_test_"
+        templates_import_filter: "^fam_tsync_test_Kickstart default$"
+        expected_change: true
+    - include_tasks: tasks/templates_import.yml
+      vars:
+        templates_import_prefix: "fam_tsync_test_"
+        templates_import_filter: "^fam_tsync_test_Kickstart default$"
+        expected_change: false
+    - include: tasks/provisioning_template.yml
+      vars:
+        provisioning_template_name: "fam_tsync_test_Kickstart default"
+        provisioning_template_template: "Modified Kickstart default provisioning template"
+        provisioning_template_state: present
+        provisioning_template_kind: provision
+        provisioning_template_locations: '{{ omit }}'
```

Comment:
There's no need, they default to `omit` anyway.
However, as i fail to see, where you resolve the taxonomies into ids, id love to see them somewhere in the tests.

Reply:
fair point and it's not too hard to add, ok

Reply:
Yupp, I think currently the imported templates won't be taxed properly.

Reply:
sounds like a plugin bug to me? document and move on? :)

Reply:
Ok, there were two problems
1. non-entity modules did not have functionality to lookup entity params and flatten them
I made a mock attempt to do this - will start a new thread when pushed
2. bug https://projects.theforeman.org/issues/29534
made the assertion for the change think "false" for now